Source/Evolve.cpp

```diff
@@ -193,6 +193,73 @@ void interpolate_rhs_from_mesh(FlavoredNeutrinoContainer& neutrinos_rhs, const M
     [=] AMREX_GPU_DEVICE (FlavoredNeutrinoContainer::ParticleType& p,
                           amrex::Array4<const amrex::Real> const& sarr)
     {
+
+        // If statement to avoid computing quantities of particles inside the black hole.
+        if( parms->do_blackhole==1 ){
+        
+            // Compute particle distance from black hole center
+            double particle_distance_from_bh_center = pow( pow( p.rdata(PIdx::x) - parms->bh_center_x , 2.0 ) + pow( p.rdata(PIdx::y) - parms->bh_center_y , 2.0 ) + pow( p.rdata(PIdx::z) - parms->bh_center_z , 2.0 ) , 0.5 ); //cm
```

Comment:
In order to calculate the square root of x, you should use `sqrt(x)` instead of `pow(x, 0.5)`. 
`sqrt()` function is faster. 
Also, if you just need to calculate the square of x, then it is better to use `x*x` instead of `pow(x,2)`. 
`pow()` function should be used only when it is absolutely necessary because it is slow.

Reply:
Here, you can just remove the `pow()` function and use `sqrt()` function instead.

Reply:
you can also use `amrex::Math::powi<2>(x)` for integer powers